README.md

```diff
@@ -1 +1,25 @@
-# Genegram
\ No newline at end of file
+# Genegram
+
+## Description
+
+[comment]: <> (TODO)
+
+## Installation
+
+Simply clone the repository and run the following commands:
+
+```bash
+pip install -r requirements.txt
+pip install .
+```
+
+## Usage
+
+Run the `python -m generam` with the arguments.
+
+### **Required arguments**
+
+Argument | Description
+:--- | :---
+--seq_fasta | Path to the `seq.fasta` file
+--predict | Path to the folder where the predictions will be saved
```

Comment:
```predict``` looks like a task (predict, filter, rebase, etc), not a path to the folder. Maybe, we can use more typical ```--out``` or something similar.